x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/esnative/ReservedRealm.java

```diff
@@ -245,6 +237,14 @@ private void getUserInfo(final String username, @Nullable SecureString credentia
         }
     }
 
+    private void getAutoconfiguredUser(final String username, SecureString credentials, ActionListener<ReservedUserInfo> listener) {
```

Comment:
Naming is hard.. does this get the/a Auto Configured user ?

Reply:
You are right, not necessary autoconfigured.  May be `getAutoconfiguredOrDefaultUser`